static/js/util/ProfileFormFields.js

```diff
@@ -60,28 +48,36 @@ export default class ProfileFormFields extends React.Component {
   }
   // type declarations
   boundTextField: Function;
-  boundSelectField: Function;
-  boundCountrySelectField: Function;
-  boundStateSelectField: Function;
   boundDateField: Function;
   boundRadioGroupField: Function;
-  countryOptions: Option[];
   genderOptions: Option[];
   languageOptions: Option[];
   privacyOptions: Array<{value: string, label: string, helper: string}>;
   educationLevelOptions: Option[];
   industryOptions: Option[];
 
+  defaultInputComponentProps = () => {
```

Comment:
can you add a type annotation here? this is the only type error I'm seeing on the branch currently.

if should look something like

```js
defaultInputComponentProps: Function = (): void => {
  ...
};
```